packages/kit/src/api/provider.tsx

```diff
@@ -23,10 +23,10 @@ import { quadraticFunding } from '../strategies/quadratic-funding';
 import { Toaster } from '../ui/toaster';
 
 if (typeof window !== 'undefined') {
-  // posthog.init('phc_MkecAopGBhofBbwLqvcvV0iyHBZWSlemr7krp6lxLjl', {
-  //   api_host: 'https://us.i.posthog.com',
-  //   person_profiles: 'always',
-  // });
+  posthog.init('phc_h8kZXpuDPeUSkSNgclmQFhKyMrgWqjwd8BDvO0CWWtE', {
```

Comment:
Move to environment bro! 😄

Reply:
it is their code :D just uncommented it :D

Reply:
I know but it makes no sense 😆

Reply:
Resolved, thx @b0rza